packages/build/README.md

```diff
@@ -7,7 +7,6 @@ LoopBack 4 or other TypeScript modules, including:
   [`tsc`](https://www.typescriptlang.org/docs/handbook/compiler-options.html) to
   compile typescript files
 - lb-eslint: Run [`eslint`](https://typescript-eslint.io/)
-- lb-tslint: Run [`tslint`](https://github.com/palantir/tslint)
```

Comment:
This is great.

In general, we try to keep commits small in scope and avoid mixing unrelated changes in one commit. Ideally, we prefer to have one pull request for each logical change.

Please:
- Extract this change into a standalone commit inside this pull request
- Optionally, move that commit into a new pull request, and keep this PR to discuss instructions for nyc. This way we can land the easy change quickly.